# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/base/CacheConstants.h
# Commit: b92893ebb007
# Full Hash: b92893ebb007f76bc8b7a9c070e2938a0598a381
# Author: James Teh <jteh@mozilla.com>
# Date: 2021-10-08 09:48:33
# Regressor Bug: 1734322
# File Overlap Count: 1
# Description:
#   Bug 1734322: Cash DOM node id in the parent process a11y cache. r=eeejay
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D127757
# ==============================================================================

diff -r 92bfb5c41eb7 -r b92893ebb007 accessible/base/CacheConstants.h
--- a/accessible/base/CacheConstants.h	Fri Oct 08 00:52:39 2021 +0000
+++ b/accessible/base/CacheConstants.h	Fri Oct 08 01:57:46 2021 +0000
@@ -17,6 +17,7 @@
   static constexpr uint64_t Bounds = ((uint64_t)0x1) << 2;
   static constexpr uint64_t Resolution = ((uint64_t)0x1) << 3;
   static constexpr uint64_t Text = ((uint64_t)0x1) << 4;
+  static constexpr uint64_t DOMNodeID = ((uint64_t)0x1) << 4;
   static constexpr uint64_t All = ~((uint64_t)0x0);
 };
 